src/pybind/mgr/orchestrator/module.py

```diff
@@ -326,6 +326,9 @@ def _list_devices(self, hostname=None, format='plain', refresh=False):
                             d.human_readable_type,
                             format_bytes(d.sys_api.get('size', 0), 5),
                             d.device_id,
+                            d.sys_api.get('model', 'n/a') or 'n/a',
```

Comment:
Note that the `default-> n/a or n/a` is on purpose. 

While the first safeguards against a `None` from the dict.get access, the seconds prints something meaningful in case there is an empty string. (often the case for disks' model attribute in libvirt environments)

Reply:
`d.sys_api.get('model') or 'n/a',` also works.

`Fixes: https://tracker.ceph.com/issues/45872` is missing in the commit message.

Reply:
> `d.sys_api.get('model') or 'n/a',` also works.

No, it doesn't :)

``` python
In [1]: {'foo': ''}.get('foo')                                                  
Out[1]: ''

In [2]: {'foo': ''}.get('foo', 'n/a')                                           
Out[2]: ''

In [3]: {'foo': ''}.get('foo', 'n/a') or 'n/a'                                  
Out[3]: 'n/a'
```

> Fixes: https://tracker.ceph.com/issues/45872 is missing in the commit message.

thanks, fixing